jetson/percep/pcl.hpp

```diff
@@ -67,17 +87,24 @@ class PCL {
         void FindInterestPoints(std::vector<pcl::PointIndices> &cluster_indices, std::vector<std::vector<int>> &interest_points);
         
         //Finds a clear path given the obstacle corners
-        double FindClearPath(std::vector<std::vector<int>> interest_points,
+        double FindClearPath(const std::vector<std::vector<int>> &interest_points,
                            shared_ptr<pcl::visualization::PCLVisualizer> viewer);
 
         //Determines whether the input path is obstructed
-        bool CheckPath(std::vector<std::vector<int>> interest_points,
+        bool CheckPath(const std::vector<std::vector<int>> &interest_points,
                shared_ptr<pcl::visualization::PCLVisualizer> viewer,
                std::vector<int> &obstacles, compareLine leftLine, compareLine rightLine);
+        
+        /**
+        \brief Determines angle off center a clear path can be found
+        \param direction: given 0 finds left clear path given 1 find right clear path
```

Comment:
What is the point of these leading slashes?

Reply:
VS Code is able to display them nicely in the comment and when you hover over the function or an instance of the function they provide you extra info ab the function. I saw Stereo Labs doing it and figured we should give it a try. I'll add it to #516 to make everything this way

![Screenshot from 2021-01-02 11-16-35](https://user-images.githubusercontent.com/49988239/103461459-827aa280-4cec-11eb-8d35-d68ad35b1ccc.png)
![Screenshot from 2021-01-02 11-16-41](https://user-images.githubusercontent.com/49988239/103461463-860e2980-4cec-11eb-9c2e-9b0d3a314629.png)

Reply:
Sounds good to me! Definitely a good idea to add to that issue then.

Reply:
It looks to me like the indentation isn't consistent across the two times you have it here. Make sure when resolving #516 you do it all the same way.